docs/api/go/examples/custom-paymaster/use-paymaster.md

```diff
@@ -35,7 +35,7 @@ import (
 func main() {
 	var (
 		PrivateKey        = os.Getenv("PRIVATE_KEY")
-		ZkSyncEraProvider = "https://testnet.era.zksync.dev"
+		ZkSyncEraProvider = "https://sepolia.era.zksync.dev"
 		TokenAddress      = common.HexToAddress("0xCd9BDa1d0FC539043D4C80103bdF4f9cb108931B") // Crown token which can be minted for free
 		PaymasterAddress  = common.HexToAddress("0xd660c2F92d3d0634e5A20f26821C43F1b09298fe") // Paymaster for Crown token
```

Comment:
This paymaster is not deployed on Sepolia

Reply:
We'll be reverting the Go SDK examples/docs back to use Goerli Testnet until we can find an appropriate path forward. Good catch!